src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -387,7 +387,8 @@ public virtual ImmutableArray<CSharpAttributeData> GetReturnTypeAttributes()
         /// ignoring any other overriding methods in base classes.
         /// </summary>
         /// <param name="accessingTypeOpt">The search must respect accessibility from this type.</param>
-        internal MethodSymbol GetLeastOverriddenMethod(NamedTypeSymbol accessingTypeOpt)
+        /// <param name="withSameReturnType">The returned method must have the same return type.</param>
+        internal MethodSymbol GetLeastOverriddenMethod(NamedTypeSymbol accessingTypeOpt, bool withSameReturnType = false)
```

Comment:
> = false [](start = 112, length = 8)

Please make this a required parameter so that every consumer has to make an explicit decision what to pass, and, while reviewing this PR, we could be certain that all existing call sites are properly adjusted.

Reply:
Parameter is removed later in this PR.

---
In reply to: [424627754](https://github.com/dotnet/roslyn/pull/44025#discussion_r424627754) [](ancestors = 424627754)